data_steward/validation/participants/validate.py

```diff
@@ -0,0 +1,126 @@
+"""
+Script to update a site's DRC identity match table with indications of matches
+between EHR submitted fields and Participant Summary API fields.
+
+There should be a record for each participant and the record should be filled with default values of `missing_rdr` or
+    `missing_ehr`. Each record should contain data for the fields: person_id, first_name, middle_name, last_name,
+    phone_number, email, address_1, address_2, city, state, zip, birth_date, sex, and algorithm.
+
+The record for each of the above fields should default to `missing_rdr` if the joined record in the
+    ps_api_values_<hpo_id> table does not contain any information otherwise, it should default to `missing_ehr`
+
+A match is indicated by a field value of 'match' and a non-match is indicated by a field values of 'non_match'
+
+Original Issue: DC-1127
+"""
+
+# Python imports
+import logging
+import argparse
+
+# Project imports
+from utils import bq, pipeline_logging, auth
+from tools.create_tier import SCOPES
+from common import JINJA_ENV, PS_API_VALUES, DRC_OPS
+from .participant_validation_queries import CREATE_COMPARISON_FUNCTION_QUERIES
+
+LOGGER = logging.getLogger(__name__)
+
+EHR_OPS = 'ehr_ops'
+MATCH = 'match'
+NO_MATCH = 'no_match'
+MISSING_RDR = 'missing_rdr'
+MISSING_EHR = 'missing_ehr'
+
+MATCH_FIELDS_QUERY = JINJA_ENV.from_string("""
+    UPDATE `{{project_id}}.{{drc_dataset_id}}.{{id_match_table_id}}` upd
+    SET upd.email = `{{project_id}}.{{drc_dataset_id}}.CompareEmail`(ps.email, ehr.email),
+        upd.algorithm = 'yes'
+    FROM `{{project_id}}.{{drc_dataset_id}}.{{ps_api_table_id}}` ps
+    LEFT JOIN `{{project_id}}.{{ehr_ops_dataset_id}}.{{hpo_pii_email_table_id}}` ehr
+        ON ehr.person_id = ps.person_id
+    WHERE upd.person_id = ps.person_id
+        AND upd._PARTITIONTIME = ps._PARTITIONTIME
+""")
+
+
+def identify_rdr_ehr_match(client, project_id, hpo_id, ehr_ops_dataset_id):
+
+    id_match_table_id = f'drc_identity_match_{hpo_id}'
```

Comment:
This might need to be just `identity_match_{hpo_id}`, we have schema for it. Is this a different table from that?

Reply:
I think they reference the same thing, but the ticket requests 'drc_identity_match'. I guess it's a rebranding?

Reply:
Thanks for updating this!